### ISSUE TITLE
Failed to load interface for Lambdabot.FixPrecedence

### ISSUE THREAD

**Issue by tfausak** [23Skidoo/pointful#4]
I'm not familiar enough with Cabal to know what's going on here.

```
$ cabal sandbox init
Writing a default package environment file to
.../cabal.sandbox.config
Creating a new sandbox at
.../.cabal-sandbox
$ cabal install pointful-1.0.5
Resolving dependencies...
Notice: installing into a sandbox located at
.../.cabal-sandbox
Configuring pointful-1.0.5...
Building pointful-1.0.5...
Installed pointful-1.0.5
$ cabal exec ghci
GHCi, version 7.8.3: http://www.haskell.org/ghc/  :? for help
Loading package ghc-prim ... linking ... done.
Loading package integer-gmp ... linking ... done. 
Loading package base ... linking ... done.
>>> import Lambdabot.Pointful
Failed to load interface for ‘Lambdabot.FixPrecedence’
Perhaps you haven't installed the "dyn" libraries for package ‘pointful-1.0.5’?
Use -v to see a list of the files searched for.
```

**Comment by 23Skidoo** [comment]
> ```
> Failed to load interface for ‘Lambdabot.FixPrecedence’
> Perhaps you haven't installed the "dyn" libraries for package ‘pointful-1.0.5’?
> ```

Try `cabal install --enable-shared --reinstall pointful-1.0.5`. Also, you can use `cabal repl` instead of `cabal exec ghci`.

**Comment by tfausak** [comment]
I get the same issue with `--enable-shared`. 

``` sh
$ ghc --version
The Glorious Glasgow Haskell Compilation System, version 7.8.4
$ cabal --version
cabal-install version 1.22.0.1
using version 1.22.1.0 of the Cabal library 
$ rm -f -r /tmp/pointful
$ mkdir /tmp/pointful
$ cd /tmp/pointful
$ cabal update
Downloading the latest package list from hackage.haskell.org
Skipping download: Local and remote files match.
$ cabal sandbox init
Writing a default package environment file to
/tmp/pointful/cabal.sandbox.config
Creating a new sandbox at /tmp/pointful/.cabal-sandbox
$ cabal install happy
Resolving dependencies...
Notice: installing into a sandbox located at /tmp/pointful/.cabal-sandbox
Configuring transformers-0.4.3.0...
Building transformers-0.4.3.0...
Installed transformers-0.4.3.0
Configuring mtl-2.2.1...
Building mtl-2.2.1...
Installed mtl-2.2.1
Configuring happy-1.19.5...
Building happy-1.19.5...
Installed happy-1.19.5
$ cabal install --enable-shared pointful-1.0.5
Resolving dependencies...
Notice: installing into a sandbox located at /tmp/pointful/.cabal-sandbox
Configuring syb-0.4.4...
Building syb-0.4.4...
Installed syb-0.4.4
Configuring haskell-src-1.0.2.0...
Building haskell-src-1.0.2.0...
Installed haskell-src-1.0.2.0
Configuring pointful-1.0.5...
Building pointful-1.0.5...
Installed pointful-1.0.5
$ echo 'import Lambdabot.Pointful' | cabal repl
GHCi, version 7.8.4: http://www.haskell.org/ghc/  :? for help
Loading package ghc-prim ... linking ... done.
Loading package integer-gmp ... linking ... done.
Loading package base ... linking ... done.
Prelude> Failed to load interface for ‘Lambdabot.FixPrecedence’
Perhaps you haven't installed the "dyn" libraries for package ‘pointful-1.0.5’?
Use -v to see a list of the files searched for.
Prelude> Leaving GHCi.
```

I think the other Lambdabot modules need to be added to `other-modules` for the library. I should have done that in #3.

**Comment by 23Skidoo** [comment]
I've updated `other-modules`.

**Comment by 23Skidoo** [comment]
Released 1.0.6.

**Comment by tfausak** [comment]
That did the trick. Thank you! Sorry for not catching this in my pull request.
